src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const textData = document.querySelectorAll('.population');
+const items = document.querySelectorAll('.list__item');
+let total = 0;
+
+textData.forEach(node => {
```

Comment:
Replace 'forEach' with 'reduce' to calculate 'total'. This way, you avoid declaring 'total' as a mutable variable.